README.md

```diff
@@ -49,7 +51,20 @@ To run the application, run both the [Todo.Web/Server](Todo.Web/Server) and [Tod
       dotnet tool install --global Microsoft.Tye --version 0.11.0-alpha.22111.1
       ```
 
-      Run `tye run` in the repository root and navigate to the tye dashboard (usually http://localhost:8080) to see both applications running.
+      Run `tye run` in the repository root and navigate to the tye dashboard (usually http://localhost:8000) to see both applications running.
+
+   - **Docker Compose** - Open your terminal, navigate to the root folder of this project and run the following commands:
+   1. Build a docker image for the `TodoApi` directly from dotnet publish.
+      ```
+      dotnet publish ./TodoApi/TodoApi.csproj --os linux --arch x64 /t:PublishContainer -c Release
+      ```
+
+   1. Build a docker image for the `Todo.Web.Server` directly from dotnet publish.
+      ```
+      dotnet publish ./Todo.Web/Server/Todo.Web.Server.csproj --os linux --arch x64 /t:PublishContainer -c Release --self-contained true
```

Comment:
The flag `--self-contained true` is needed here due to this open issue [Publishing hosted Blazor WASM project breaks if not self-contained](https://github.com/dotnet/runtime/issues/75847).